tests/integration/Core/Content/Mail/Service/EmailSenderTest.php

```diff
@@ -46,7 +46,9 @@ private function doRunTest(): void
         $subject = 'mail create test';
         $sender = ['testSender@example.org' => 'Sales Channel'];
         $recipients = ['testReceiver@example.org' => 'Receiver name', 'null-name@example.org' => null];
-        $contents = ['text/html' => 'Message'];
+        // We need a large text, otherwise the mail will not be stored in the filesystem, but sent directly
+        $text = str_repeat('a', 1024 * 256 + 1);
```

Comment:
shouldn't we use the constant here? if the value of the constant will change, the test will fail